pkg/reconciler/pipelinerun/resources/pipelinerunresolution_test.go

```diff
@@ -2520,23 +2500,19 @@ func TestValidateResourceBindings_Missing(t *testing.T) {
 		Spec: v1beta1.PipelineSpec{
 			Resources: []v1beta1.PipelineDeclaredResource{
 				gitDeclaredResource,
-				{
-					Name: "image-resource",
-					Type: resourcev1alpha1.PipelineResourceTypeImage,
-				},
 			},
 		},
 	}
 	pr := &v1beta1.PipelineRun{
 		ObjectMeta: metav1.ObjectMeta{Name: "pipelinerun"},
 		Spec: v1beta1.PipelineRunSpec{
 			PipelineRef: &v1beta1.PipelineRef{Name: "pipeline"},
-			Resources:   []v1beta1.PipelineResourceBinding{gitSweetResourceBinding},
+			Resources:   []v1beta1.PipelineResourceBinding{},
```

Comment:
why remove this?

Reply:
In this case the image-resource was the missing one but now since it is being removed, we would have to change to the other resource that should be missing but required by the pipeline resource declaration.